tool/resident.c

```diff
@@ -0,0 +1,26 @@
+#include "tool.h"
+
+/**
+ * Setup a resident data block for our TSR version.
+ * See: https://forum.vcfed.org/index.php?threads/tsr-programs-with-open-watcom.23586/ for details.
+ */
+//#pragma data_seg("BEGTEXT", "CODE")
+//#pragma code_seg("BEGTEXT", "CODE")
+#pragma data_seg("BEGTEXT", "BEGDATA")
```

Comment:
In theory, this keeps forces the data to be as early as possible in the application.